src/Core/src/Handlers/Button/ButtonHandler.Windows.cs

```diff
@@ -108,6 +111,24 @@ void OnPointerPressed(object sender, PointerRoutedEventArgs e)
 		void OnPointerReleased(object sender, PointerRoutedEventArgs e)
 		{
 			VirtualView?.Released();
+
+			// In Windows pressing the button sets the focus. This causes the VisualState to go from Pressed to Focused
+			// while on the rest of supported platforms, it goes to Normal.
+			// Here we force the same behavior as in the rest of the platforms but we keep the possibility of setting the focus with Keyboard, etc.
+			UpdateVisualState();
+		}
+
+		void UpdateVisualState()
+		{
+			if (PlatformView is not null)
+			{
+				var dispatcher = MauiContext?.Services.GetRequiredService<IDispatcher>();
```

Comment:
If MauiContext is null, the we have Services as null and then GetRequiredService will throw a NRE, correct?

Reply:
Right, added protection to this posible case.